lib/resources.js

```diff
@@ -259,8 +259,6 @@ module.exports = function (app, _private = null) {
     app.logger.debug('Resources#itemsByFilter', body)
     return app.esClient.search(body)
       .then((resp) => {
-        resp = resp.body
```

Comment:
what made this change possible?

Reply:
Subtle API change in the move from Elasticsearch client v7 to v8